src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors/codeSmell/UsageOfSilenceOperatorInspector.java

```diff
@@ -33,8 +34,8 @@
  */
 
 public class UsageOfSilenceOperatorInspector extends BasePhpInspection {
-    // configuration flags automatically saved by IDE
-    public boolean RESPECT_CONTEXT = true;
+    // Inspection options.
+    public boolean optionRespectContext = false;
```

Comment:
@rentalhost : this is really needed. I need a good analogy, imagine that yo was tweaking IDE code style (e.g. align assignments, + 25 other options you are not remembering anymore and) and PS on update did reset them.

I don't want our community to run into this. The change is a good one, but we can not afford it, pleaseeeeeeeeeeee return naming back.